docs/reference/ilm/actions/ilm-shrink.asciidoc

```diff
@@ -103,3 +91,25 @@ PUT _ilm/policy/my_policy
   }
 }
 --------------------------------------------------
+
+[[ilm-shrink-retry-failed-allocation]]
+==== Retrying failed allocation steps
```

Comment:
@jrodewig the threshold and retry-ability/rewind of the shrink action is based on waiting for some conditions that are not fulfilled until the threshold is breached/surpassed. Should we be focused more on the "wait until threshold and then rewind" as opposed to retrying a step?

Reply:
Good point. I renamed this "Shard allocation for shrink" as it could also be useful for people curious about how shrink allocates normally.